src/protocols/secure_channel/CASESession.cpp

```diff
@@ -308,16 +324,17 @@ CHIP_ERROR CASESession::DeriveSecureSession(CryptoContext & session, CryptoConte
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR CASESession::SendSigmaR1()
+CHIP_ERROR CASESession::SendSigma1()
 {
-    size_t data_len =
-        EstimateTLVStructOverhead(kSigmaParamRandomNumberSize + sizeof(uint16_t) + kSHA256_Hash_Length + kP256_PublicKey_Length, 4);
+    size_t data_len = EstimateTLVStructOverhead(kSigmaParamRandomNumberSize + sizeof(uint16_t) + kSHA256_Hash_Length +
+                                                    kP256_PublicKey_Length + kCASEResumptionIDSize + kTAGSize,
+                                                7);
```

Comment:
Suggest adding `/* + MRPOptionalParams */` to the mix and a TODO to add MRP optional  params. That accounts for the 7. Otherwise there are 6 fields present and we are estimating for 7 (for the future). I would want to avoid someone bumping 7->8 when MRP params arrive.